### PR TITLE
Added Additonal Tip

### PR DESCRIPTION
**Pull request by Gourmand-is-OP**
#### Summary
SUMMARY: Content "Added additional tip"
Added tip: "Don't be afraid to make the game easier on yourself. Worlds can be made to spawn with more resources."

### CODE REVIEW

**Comment by alanbrady** [comment]
Jenkins rebuild

File: data/core/tips.json [hunk 1]
```diff
@@ -283,5 +283,10 @@
     "type": "snippet",
     "category": "tip",
     "text": [ "Broken limb pinned you down?  Hospital might have next gen tech to get you going." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Don't be afraid to make the game easier on yourself. Worlds can be made to spawn with more resources." ]
```
Comment (alanbrady) [inline data/core/tips.json]: Two spaces after period here. `... yourself.  Worlds```

Comment (Gourmand-is-OP) [reply data/core/tips.json]: Has been edited.

**Comment by alanbrady** [comment]
Thanks!  Seems like a reasonable tip and things seem good otherwise.

**Comment by lispcoc** [comment]
VERY good. C:DDA today is too hard to play for begginers...

**Pull request merged**